src/Orleans/Statistics/SchedulerStatisticsGroup.cs

```diff
@@ -36,7 +37,7 @@ internal class SchedulerStatisticsGroup
         private static object lockable;
         private static Logger logger;
 
-        internal static void Init()
+        internal static void Init(ILoggerFactory loggerFactory)
```

Comment:
hmmm, this is very bad, as we'll leak the factory/logger into a static field and will survive event after the silo/client goes away. If not addressing right away, log it as an issue we need to fix, similar to the `UnobservedExceptionsHandlerClass` case

Reply:
issue created in #3429